addons/delivery/migrations/9.0.1.0/post-migration.py

```diff
@@ -26,6 +26,26 @@ def rename_property(cr, model, old_name, new_name):
         (new_name, field_ids))
 
 
+def reassign_carrier_id(cr):
+    cr.execute("""
+        select id, openupgrade_legacy_9_0_carrier_id from delivery_carrier 
+        where id != openupgrade_legacy_9_0_carrier_id; 
+    """)
+    for new_id, old_id in cr.fetchall():
+        if openupgrade.table_exists(cr, 'sale_order') and \
+                openupgrade.column_exists(cr, 'sale_order', 'carrier_id'):
+            cr.execute("""
+                UPDATE sale_order set carrier_id = %s 
+                where carrier_id = %s
+            """ % (new_id, old_id))
+        if openupgrade.table_exists(cr, 'stock_picking') and \
+                openupgrade.column_exists(cr, 'stock_picking', 'carrier_id'):
```

Comment:
line too long

Reply:
the length is 78 characters, is that bad?

Reply:
ah sorry, I thought that was the travis error 😅